public/wp-content/themes/coloredcow/src/scss/header/style.scss

```diff
@@ -1,6 +1,6 @@
 
 .navbar-toggle{
-	@extend .ml-auto;
+	@extend .ms-auto !optional;
```

Comment:
is optional still needed?

Reply:
No we don't need that, I have removed the same in the latest commit.